crates/anvil/core/src/eth/utils.rs

```diff
@@ -6,7 +6,7 @@ use ethers_core::{
     },
 };
 use foundry_evm::utils::{h160_to_b160, h256_to_u256_be, u256_to_ru256};
-use revm::primitives::{B160, U256 as rU256};
+use revm::primitives::{Address as rAddress, U256 as rU256};
```

Comment:
I would replace all `revm::primitives` usage with `alloy_primitives` - these should be the same type now, right?

Reply:
Done on https://github.com/foundry-rs/foundry/pull/5768/commits/69e06eed172cb708ca042e8266b6f9c64a96ac01